java-codegen/src/main/resources/org/opensearch/client/codegen/templates/ObjectShape/HashCode.mustache

```diff
@@ -0,0 +1,17 @@
+    public int hashCode() {
+        int result = 17;
+        {{#fields}}
```

Comment:
This needs to also account for `super.hashCode()`

Reply:
done as part of [8fd4694](https://github.com/opensearch-project/opensearch-java/pull/1201/commits/8fd4694c6f4952e1ba2b18453b8603e23e74b514)